translations/en/global_indicators.yml

```diff
@@ -20,28 +20,32 @@
   strategies in line with the Sendai Framework for Disaster Risk Reduction 2015–2030
 1-5-4-title: Proportion of local governments that adopt and implement local disaster
   risk reduction strategies in line with national disaster risk reduction strategies
-1-a-1-title: Proportion of domestically generated resources allocated by the government
-  directly to poverty reduction programmes
+1-a-1-title: Total official development assistance grants from all donors that focus
+  on poverty reduction as a share of the recipient country’s gross national income
 1-a-2-title: Proportion of total government spending on essential services (education,
   health and social protection)
 1-a-3-title: Sum of total grants and non-debt-creating inflows directly allocated
   to poverty reduction programmes as a proportion of GDP
-1-b-1-title: Proportion of government recurrent and capital spending to sectors that
-  disproportionately benefit women, the poor and vulnerable groups
+1-b-1-title: Pro-poor public social spending
 10-1-1-title: Growth rates of household expenditure or income per capita among the
   bottom 40 per cent of the population and the total population
 10-2-1-title: Proportion of people living below 50 per cent of median income, by sex,
   age and persons with disabilities
 10-3-1-title: Proportion of population reporting having personally felt discriminated
   against or harassed in the previous 12 months on the basis of a ground of discrimination
   prohibited under international human rights law
-10-4-1-title: Labour share of GDP, comprising wages and social protection transfers
-10-5-1-title: Financial Soundness Indicators (FSIs)
+10-4-1-title: Labour share of GDP
+10-4-2-title: Redistributive impact of fiscal policy4
```

Comment:
@brockfanning 4.c.1 is now ok but I can see there's still a number at the end of 10.4.2 - there may be other's but this one has been pointed out to me by data team